src/api/component.js

```diff
@@ -194,81 +133,66 @@ Component.updated = function updated (elem, prev) {
       return true;
     }
   }
+
   return false;
 };
 
-// Skate
-//
-// Calls the user-defined updated() lifecycle callback.
-Component[$updated] = function _updated (elem) {
-  if (typeof this.updated === 'function') {
-    const prev = elem[$props];
-    elem[$props] = getSetProps(elem);
-    return this.updated(elem, prev);
-  }
-  return true;
+// DEPRECATED
+Component.rendered = function _rendered () {};
```

Comment:
That's a good point. I think that's a more involved discussion in terms of how do we do it / what do we use. It'd have to be ignored in node prod etc. Happy to issue a second pr for this along side it (just so it doesn't block this going in). Any ideas?